File: jambeth/jAmbeth.Xml/src/de/osthus/ambeth/config/XmlConfigurationConstants.java
package de.osthus.ambeth.config;

import de.osthus.ambeth.annotation.ConfigurationConstantDescription;
import de.osthus.ambeth.annotation.ConfigurationConstants;

@ConfigurationConstants
public final class XmlConfigurationConstants
{
	@ConfigurationConstantDescription("TODO")
	public static final String PackageScanPatterns = "ambeth.xml.transfer.pattern";

	private XmlConfigurationConstants()
	{
		// Intended blank
	}
}

File: ambeth/Ambeth.Xml/ambeth/config/XmlConfigurationConstants.cs
using De.Osthus.Ambeth.Annotation;
using System;

namespace De.Osthus.Ambeth.Config
{
    [ConfigurationConstants]
    public sealed class XmlConfigurationConstants
    {
        public const String PackageScanPatterns = "ambeth.xml.transfer.pattern";

        private XmlConfigurationConstants()
        {
            // Intended blank
        }
    }
}
